src/plugins/safeSnap/utils/umaModule.ts

```diff
@@ -70,12 +71,65 @@ export const queryGql = async (url: string, query: string) => {
         `GraphQL Error: ${data.errors.map(error => error.message).join(', ')}`
       );
     }
-    return data.data;
+    return data.data as Result;
   } catch (error) {
     throw new Error(`Network error: ${error.message}`);
   }
 };
 
+export const getSpaceHasOsnapEnabledTreasury = async (treasuries: TreasuryWallet[]) => {
+  const isOsnapEnabledOnTreasuriesResult = await Promise.all(treasuries.map(treasury => getIsOsnapEnabled(treasury.network, treasury.address)))
+  return isOsnapEnabledOnTreasuriesResult.some(isOsnapEnabled => isOsnapEnabled);
+}
+
+export const getIsOsnapEnabled = async (
+  network: string,
+  safeAddress: string
+) => {
+  const subgraph = getOptimisticGovernorSubgraph(network);
+  const query = `
+      query isOSnapEnabled {
+        safe(id:"${safeAddress.toLowerCase()}"){
+          isOptimisticGovernorEnabled
+        }
+      }
+    `;
+  type Result = {
+    safe: { isOptimisticGovernorEnabled: boolean };
+  };
+  const result = await queryGql<Result>(subgraph, query);
+  return result?.safe?.isOptimisticGovernorEnabled ?? false;
+};
+
+export function makeConfigureOsnapUrl(params: {
```

Comment:
do we need these functions in the safe snap app, or is it something we can remove?

Reply:
lets remove, I added this before we fully decided to go with our own plugin